mbl-core/tests/partitions/test_partitions.py

```diff
@@ -126,5 +127,22 @@ def get_actual_part_table():
     return part_table
 
 
+def get_var_for_actual_part(var_name, part_name, default=None):
```

Comment:
`part_name` is actually a path something like `/sys/block/mmcblk1/mmcblk1p2`, right?

Can you rename the param to something more descriptive please (and update the docstring)? I can't think of a great name... Something like `sys_part_info_path`?

Reply:
Changed the parameters to the method.